sklearn/model_selection/_split.py

```diff
@@ -1194,22 +1196,17 @@ class RepeatedKFold(_RepeatedSplits):
     >>> from sklearn.model_selection import RepeatedKFold
     >>> X = np.array([[1, 2], [3, 4], [1, 2], [3, 4]])
     >>> y = np.array([0, 0, 1, 1])
-    >>> rkf = RepeatedKFold(n_splits=2, n_repeats=2, random_state=2652124)
+    >>> rkf = RepeatedKFold(n_splits=2, n_repeats=2, random_state=2652123)
```

Comment:
O_o what's so especial about this number

Reply:
with the previous seed 2652124 the repeated folds where the same which was bad at illustrating the purpose of `RepeatedKFold`